package.json

```diff
@@ -16,7 +16,7 @@
     "prettier": "prettier --write 'src/**/*.js'",
     "lint": "eslint src",
     "flow": "flow src",
-    "jest": "rm -rf libcoredb && mkdir libcoredb && cross-env TZ=America/New_York jest",
+    "jest": "npx rimraf libcoredb && mkdir libcoredb && cross-env TZ=America/New_York jest",
```

Comment:
better be a dev deps to not being installed each time we run.

Reply:
ok